tests/test_utils.py

```diff
@@ -188,3 +190,16 @@ def test_consume():
     consume(x)
     with pytest.raises(StopIteration):
         next(x)
+
+
+class TestNamespace:
+    def test_isawaitable(self):
+        """Test async compatibility."""
+
+        async def _test():
+            environment = Environment(enable_async=True)
+            template = '{% set ns = namespace(foo="Bar") %}{{ ns.foo }}'
+            actual = await environment.from_string(template).render_async()
+            assert "Bar" == actual
```

Comment:
No yoda style please ;) => `assert actual == "Bar"`

Reply:
What you ask, I have done.